### PR TITLE
Move the smoke tests to this repo

### PR DESCRIPTION
**Pull request by zheileman**
Previously the smoke tests had their own repo, but moving forward we
are going to have them as part of this repo in the `smoke-tests` dir,
and eventually, will run as part of the CI.

https://www.pivotaltracker.com/story/show/150475465

### CODE REVIEW

File: smoke-tests/.env.example [hunk 1]
```diff
@@ -0,0 +1,5 @@
+TARGET_URI=http://52.demo.intranet.dsd.io/
```
Comment (tatyree) [inline smoke-tests/.env.example]: Can we change the default target?  http://intranet.docker?

File: smoke-tests/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,23 @@
+FROM ruby:2.4.1-onbuild
+
+ENV TARGET_URI http://52.demo.intranet.dsd.io/
```
Comment (tatyree) [inline smoke-tests/Dockerfile]: Can we change the default target? http://intranet.docker?

File: smoke-tests/README.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+```bash
+#!/bin/bash
+
+set -euo pipefail
+
+docker build -f Dockerfile.smoketests -t smoketests .
+docker run smoketests
+```
+
+The default target environment is `http://52.demo.intranet.dsd.io/`. You
```
Comment (tatyree) [inline smoke-tests/README.md]: Can we change the default target? http://intranet.docker?

File: smoke-tests/docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,6 @@
+version: '3'
+services:
+  smoketests:
+    build: .
+    environment:
+      - TARGET_URI=http://52.demo.intranet.dsd.io/
```
Comment (tatyree) [inline smoke-tests/docker-compose.yml]: Can we change the default target? http://intranet.docker?

**Comment by zheileman** [comment]
Sure, I'll change those URL.

**Pull request merged**